examples/example_config

```diff
@@ -0,0 +1,2 @@
+management_listener = "0.0.0.0:3444"
+ingest_listener = "0.0.0.0:3001"
```

Comment:
there is a `soldr.example.toml` file already so my PR puts these there. maybe we should move that entire file into examples directory?